validation/open_boundaries/cylinder.jl

```diff
@@ -1,127 +1,210 @@
-# This validation script shows open boundaries working in a simple case where the
-# flow remains largely unidirectional and so at one end we have no matching scheme
-# but just prescribe the inflow. At the other end we then make no assumptions about
-# the flow and use a very simple open boundary condition to permit information to 
-# exit the domain. If, for example, the flow at the prescribed boundary was reversed
-# then the model would likely fail.
-
-using Oceananigans, CairoMakie
+using Oceananigans
+using Oceananigans.Models.NonhydrostaticModels: ConjugateGradientPoissonSolver
+using Oceananigans.Solvers: DiagonallyDominantPreconditioner
+using Oceananigans.Operators: ℑxyᶠᶜᵃ, ℑxyᶜᶠᵃ
+using Oceananigans.Solvers: FFTBasedPoissonSolver
+using Printf
+using CUDA
+
 using Oceananigans.BoundaryConditions: FlatExtrapolationOpenBoundaryCondition
+                                       PerturbationAdvectionOpenBoundaryCondition
+
+# there is some problem using ConjugateGradientPoissonSolver with TimeInterval because the timestep can go really small
+# so while I identify the issue I'm using IterationInterval and a fixed timestep
```

Comment:
Have you tried setting `minimum_relative_step`? (see https://github.com/CliMA/Oceananigans.jl/pull/3606)

Reply:
This seems to fix the issue! I hadn't tried this since early november/december so I hadn't seen you'd made this fix so this is great, thank you

Reply:
I'll update the validation script